test/extended/apiserver/resiliency.go

```diff
@@ -69,19 +68,23 @@ var _ = ginkgo.Describe("[Conformance][sig-sno][Serial] Cluster", func() {
 		forceApiRollout(oc)
 
 		ginkgo.By("Expecting API to become unavailable")
-		wait.PollImmediate(time.Second, time.Minute, func() (bool, error) {
-			ready, _, err := isApiReady(config, httpClient)
-			return !ready, err
+		err = wait.PollImmediate(time.Second, 5*time.Minute, func() (bool, error) {
```

Comment:
why is this increased to 5 minutes?

Reply:
Turns out that it takes slightly more than a minute for the API server to go down after a rollout is triggered, and is the reason this test has been wrecking havoc on the tests following it - we never checked the return value of PollImmediate so the subsequent PollImmediate waiting for the API server to come back up would finish immediately (as it never went down in the first place, yet) then the API server would go down while the following tests execute

Reply:
Sometimes it takes more than 1 minute for API server to stop responding (tested empirically in a single node cluster, measured over 70s on a regular basis). You want to catch that to test availability afterwards.

Reply:
@mfojtik as [@Omer](https://coreos.slack.com/team/U01DQKJCMCM) responded to highlight that the API server often takes longer than 1 minute to "go  down", this point also supported by @pacevedom , we need a longer time-out.

Whereas 5 minutes may be overkill, perhaps this should be 2 minutes or 3 minutes to prevent flakiness?

What are your thoughts?